presidio-analyzer/tests/test_aba_routing_recognizer.py

```diff
@@ -17,6 +17,7 @@ def entities():
 @pytest.mark.parametrize(
     "text, expected_len, expected_positions, expected_score",
     [
+        # fmt: off
```

Comment:
Why are these non-compliant? just curious

Reply:
@omri374 it is, black completely ruins readability
![image](https://user-images.githubusercontent.com/15013757/104852641-d4364600-5904-11eb-90eb-97db4e216feb.png)
vs
![image](https://user-images.githubusercontent.com/15013757/104852647-de584480-5904-11eb-833e-43b2eb385593.png)
Also take a look on the teams channel for more decisions / questions